<?xml version="1.0"?>
<launch>
	<include file="$(find youbot_driver_ros_interface)/launch/youbot_driver.launch" />
	<include file="$(find ds4_driver)/launch/ds4_driver.launch" />
	<include file="$(find youbot_moveit)/launch/move_group.launch" />
	
	
	<node pkg="youbot_ux" name="status_to_joy" type="status_to_joy.py"/>
	<node pkg="youbot_ux" name="youbot_states" type="youbot_states.py"/>
	<node pkg="youbot_ux" name="youbot_states_feedback" type="youbot_states_feedback.py" output="screen"/>
	<node pkg="temoto_er_manager" name="temoto_er_manager" type="temoto_er_manager.cpp"/>
	<node pkg="youbot_ux" name="er_client_node" type="er_client_node.cpp" output="screen"/>
	<include file="$(find trajectory_recorder)/launch/trajectory_recorder.launch" />
	<include file="$(find trajectory_replayer)/launch/trajectory_replayer.launch" />
	<node pkg="youbot_ux" name="youbot_trajectory_record" type="youbot_trajectory_record.py"/>
	
	
</launch>

